gtdynamics/kinematics/KinematicsSlice.cpp

```diff
@@ -59,13 +76,30 @@ NonlinearFactorGraph Kinematics::pointGoalObjectives<Slice>(
   // Add objectives.
   for (const ContactGoal& goal : contact_goals) {
     const gtsam::Key pose_key = internal::PoseKey(goal.link()->id(), slice.k);
-    graph.emplace_shared<PointGoalFactor>(pose_key, p_->g_cost_model,
-                                          goal.contactInCoM(), goal.goal_point);
+    graph.add(PointGoalFactor(pose_key, p_->g_cost_model,
```

Comment:
No, it really should be emplace_shared, I think :-) why add an extra copy?

Reply:
Done